Stack/Opc.Ua.Bindings.Https/Stack/Https/HttpsTransportChannel.cs

```diff
@@ -11,18 +11,36 @@
 */
 
 
-#if !NO_HTTPS
-
 using System;
 using System.IO;
-using System.Runtime.InteropServices;
-using System.Threading.Tasks;
 using System.Net.Http;
 using System.Net.Http.Headers;
+using System.Runtime.InteropServices;
+using System.Threading.Tasks;
 
 namespace Opc.Ua.Bindings
 {
 
+    /// <summary>
+    /// Creates a new HttpsTransportChannel with ITransportChannel interface.
+    /// </summary>
+    public class UaHttpsTransportChannelFactory : ITransportChannelBinding
```

Comment:
Rename interface to ITransportChannelFactory ?

Reply:
better it is